src/alfasim_sdk/constants.py

```diff
@@ -0,0 +1,361 @@
+from enum import Enum
+
+GAS_PHASE = "gas"
+"""
+Constant to identify the gas phase
+"""
+
+OIL_PHASE = "oil"
+"""
+Constant to identify the oil phase
+"""
+
+WATER_PHASE = "water"
+"""
+Constant to identify the water phase
+"""
+
+SOLID_PHASE = "solid"
+
+GAS_FIELD = "gas"
+OIL_FIELD = "oil"
+WATER_FIELD = "water"
+WATER_DROPLET_IN_OIL_FIELD = "water_in_oil_droplet"
+DROPLET_FIELD = "droplet"
+BUBBLE_FIELD = "bubble"
+
```

Comment:
I can't remember right.
Won't those dispersed field names be deprecated in favor of names like `"{phase}_in_{layer}"`?

Reply:
I don't know =/
These constants were introduced by @tarcisiofischer  a long time ago.
https://github.com/ESSS/alfasim-sdk/commit/b2fcb420f52aaaaf3e9ccc6803c38e070b394b03

You are only seeing these lines being added because I moved the files to an src layout

Reply:
AFAIK, there is a task to tackle that to be as @prusse-martin mentioned, but today, we're still stucked with those I guess...